src/yz_events.erl

```diff
@@ -106,12 +148,24 @@ terminate(_Reason, _S) ->
 %%% Private
 %%%===================================================================
 
+%% @doc Called by {@link yz_general_sup} to create the p ETS table used to
+%% track registered events. Created when we add the handler after the supervisor
+%% is already up for yz_events.
+-spec create_table() -> ok.
+create_table() ->
+    _ = ets:new(?ETS, [named_table, public, set,
```

Comment:
As far as I can tell, all of the functions in this module that use ?ETS are called within the context of the yz_events process.  Can we therefore make this table private?